packages/core/src/app/shipping/shippingOption/MultiShippingOptionsListItemV2.tsx

```diff
@@ -22,29 +22,22 @@ export const MultiShippingOptionsListItemV2: FunctionComponent<
         </>
     );
 
-    const selectThisOption = useCallback(() => {
```

Comment:
> Selecting a shipping method for one consignment no longer updates the shipping method of another consignment.

This FE bug has two casues:
 1. `useCallback` here may return another consignment's `handleSelect`, due to `consignment` is not on the dependency list.
 2. `<RadioInput />` was in `<button />`. As a result, clicking `<RadioInput />` may trigger `<button />`'c `onClick` event twice.

Reply:
Good catch 👏